bitz0x75/Twitter-Lite-Frontend
# #165380051 Setup react part of the project

### What is this PR supposed to do?*
Create the initial setup for the react app
### How should this be manually tested?
- checkout this branch
- run `npm install` then `npm start`
- ensure the app runs locally
### Relevant PT stories
[#165380051](https://www.pivotaltracker.com/story/show/165380051)
